scripts/linters/pylint_extensions.py

```diff
@@ -141,9 +148,11 @@ def visit_call(self, node):
         num_positional_args_unused = num_positional_args
         # Check that all parameters with a default value have
```

Comment:
Please move this comment below the list variable that you have declared. It looks off.

Reply:
Done.